test/BindTest.cpp

```diff
@@ -0,0 +1,10 @@
+//
+// Created by johannes on 19.04.20.
+//
+
+#include <gtest/gtest.h>
+#include "../src/engine/Bind.h"
+
+TEST(BindTest, Bind) {
+  // Bind b;
```

Comment:
OK, since the current implementation is preliminary. But beware: Provisorien halten immer am längsten